lib/event-emit.js

```diff
@@ -22,19 +24,28 @@ function Emitter(geojson, updateInterval) {
         speeds[i] !== 0) speedmode = false;
   }
   var place = new EmitterPlace(geojson, speedmode);
+  var stepsTaken = 0;
+
+  this.setTime = function(start) {
+    stepsTaken = start / updateInterval;
+    return stepsTaken;
+  };
 
   /**
    * Return the next location event for a route.
    *
+   * @param {Number} optional stepsTaken value for seeking purposes.
```

Comment:
is stepsTaken intended to be a parameter of the `next()` function? Or are you using it globally?

e.g. `this.next = function(stepsTaken)` vs `this.next = function()`

I think including it in documentation here ^ implies the first.